src/pages/index.tsx

```diff
@@ -121,7 +135,15 @@ export default function Home() {
         >
           Search
         </button>
-        <p className="mt-2">Label: {topic}</p>
+        <p className="mt-2">
```

Comment:
Add a background and a border radius
```suggestion
        <p className="mt-2  p-2 rounded-3xl bg-slate-500 w-1/2">
```